spring-cloud-starter-dataflow-server/src/test/resources/org/springframework/cloud/dataflow/server/single/security/oauthConfig.yml

```diff
@@ -11,24 +11,40 @@ spring:
     dataflow:
       security:
         authorization:
-          map-oauth-scopes: true
-          role-mappings:
-            ROLE_VIEW: dataflow.view
-            ROLE_CREATE: dataflow.create
-            ROLE_MANAGE: dataflow.manage
-            ROLE_DEPLOY: dataflow.create
-            ROLE_DESTROY: dataflow.create
-            ROLE_MODIFY: dataflow.create
-            ROLE_SCHEDULE: dataflow.create
-security:
-  oauth2:
-    client:
-      client-id: myclient
-      client-secret: mysecret
-      access-token-uri: http://127.0.0.1:${oauth2.port}/oauth/token
-      user-authorization-uri: http://127.0.0.1:${oauth2.port}/oauth/authorize
-    resource:
-      user-info-uri: http://127.0.0.1:${oauth2.port}/me
-      token-info-uri: http://127.0.0.1:${oauth2.port}/oauth/check_token
-    authorization:
-      check-token-access: isAuthenticated()
+          provider-role-mappings:
+            uaa:
+              map-oauth-scopes: true
+              role-mappings:
+                ROLE_VIEW: dataflow.view
+                ROLE_CREATE: dataflow.create
+                ROLE_MANAGE: dataflow.manage
+                ROLE_DEPLOY: dataflow.create
+                ROLE_DESTROY: dataflow.create
+                ROLE_MODIFY: dataflow.create
+                ROLE_SCHEDULE: dataflow.create
+  security:
```

Comment:
This is another important property hierarchical change as well. We need to be explicit about this in the documentation.